.ruff.toml

```diff
@@ -0,0 +1,4 @@
+extend-select = ["I"]
+
+[isort]
+combine-as-imports = true
```

Comment:
I'd rather not set this value. We didn't set this config value before and I don't see a good reason to deviate from the default `ruff` settings.